### PR TITLE
Add annotation mode

### PR DESCRIPTION
**Pull request by TomDijkema**
Adds the annotation mode to the Digital Specimen page to allow for direct targeting of properties from the view.
This includes some slight changes and fixes to the annotation wizard in general.

### CODE REVIEW

**Review by djangobrunink — APPROVED** [review]
lgtm

File: src/components/digitalSpecimen/DigitalSpecimen.tsx [hunk 1]
```diff
@@ -99,6 +100,19 @@ const DigitalSpecimen = () => {
         }
     });
 
+    /**
+     * Function to set the annotation target state
+     * @param annotationTargetType The type of the annotation target, either class or term
+     * @param jsonPath The JSON path that targets the class or term
+     */
+    const SetAnnotationTarget = (annotationTargetType: 'class' | 'term', jsonPath: string) => {
```
Comment (southeo) [inline src/components/digitalSpecimen/DigitalSpecimen.tsx:108]: A lot of this seems duplicate from the media.tsx file, do you think you can reuse some code?

**Review by southeo — APPROVED** [review]
One comment but isn;t blocking, something to consider in a future pr
:coat:

File: src/components/digitalSpecimen/DigitalSpecimen.tsx (hunk 1 above)
Comment (TomDijkema) [reply src/components/digitalSpecimen/DigitalSpecimen.tsx:108]: Yeah my initial idea was to include as an utility function, only I am not allowed to use hooks out of React components, hence I chose to do it this way. I could include it at an ever higher level and pass it as props, but its irrelevant for a lot of child components. For now this seemed the best way, but can probably be optimized somehow in the future.

**Pull request merged**